ibm_mq/tests/README.md

```diff
@@ -23,6 +23,39 @@ Set up the following env vars in your test run configuration:
 * DYLD_LIBRARY_PATH=~/IBM-MQ-Toolkit-Mac-x64-9.1.2.0/lib64
 * IBM_MQ_VERSION= 8 or 9
 
+Development Tips
+================
+
+### How to connect to IBM MQ console
+
+```
+$ ddev env start ibm_mq py27-8
+$ docker exec -it ibm_mq bash
```

Comment:
Does that work though?
```suggestion
$ docker exec -it ibm_mq runmqsc
```

Reply:
Yes, it does, thx for the suggestion :)

Simplified the commands.